docs/instrumentation-list.yaml

```diff
@@ -8297,34 +8313,272 @@ libraries:
       - org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:[1.3.9,)
   ktor:
   - name: ktor-1.0
+    description: This standalone instrumentation enables HTTP server spans and HTTP
+      server metrics for the Ktor server.
+    semantic_conventions:
+    - HTTP_SERVER_SPANS
+    - HTTP_SERVER_METRICS
+    library_link: https://ktor.io/
+    features:
+    - HTTP_ROUTE
     source_path: instrumentation/ktor/ktor-1.0
     scope:
       name: io.opentelemetry.ktor-1.0
+      schema_url: https://opentelemetry.io/schemas/1.37.0
     target_versions:
       library:
       - io.ktor:ktor-server-core:[1.0.0,1.+)
+    telemetry:
+    - when: default
+      metrics:
+      - name: http.server.request.duration
+        description: Duration of HTTP server requests.
+        type: HISTOGRAM
+        unit: s
+        attributes:
+        - name: http.request.method
+          type: STRING
+        - name: http.response.status_code
+          type: LONG
+        - name: http.route
+          type: STRING
+        - name: network.protocol.version
+          type: STRING
+        - name: url.scheme
+          type: STRING
+      spans:
+      - span_kind: CONSUMER
+        attributes:
+        - name: client.address
+          type: STRING
+        - name: http.request.method
+          type: STRING
+        - name: http.response.status_code
+          type: LONG
+        - name: network.peer.address
+          type: STRING
+        - name: network.protocol.version
+          type: STRING
+        - name: server.address
+          type: STRING
+        - name: server.port
+          type: LONG
+        - name: url.path
+          type: STRING
+        - name: url.query
+          type: STRING
+        - name: url.scheme
+          type: STRING
+        - name: user_agent.original
+          type: STRING
+      - span_kind: SERVER
+        attributes:
+        - name: client.address
+          type: STRING
+        - name: error.type
+          type: STRING
+        - name: http.request.method
+          type: STRING
+        - name: http.request.method_original
+          type: STRING
+        - name: http.response.status_code
+          type: LONG
+        - name: http.route
+          type: STRING
+        - name: network.peer.address
+          type: STRING
+        - name: network.protocol.version
+          type: STRING
+        - name: server.address
+          type: STRING
+        - name: server.port
+          type: LONG
+        - name: url.path
+          type: STRING
+        - name: url.query
+          type: STRING
+        - name: url.scheme
+          type: STRING
+        - name: user_agent.original
+          type: STRING
   - name: ktor-2.0
+    description: |
+      This instrumentation enables HTTP client spans and HTTP client metrics for the Ktor HTTP client, and enriches HTTP server spans with route information for the Ktor server.
+    semantic_conventions:
+    - HTTP_CLIENT_SPANS
```

Comment:
doesn't it also create server spans?

Reply:
thanks, i got thrown off course because the javaagent server spans are suppressed due to already existing netty server spans. I've updated 2.0 and 3.0 to use the library tests instead